docs/content/start/telemetry.md

```diff
@@ -0,0 +1,255 @@
+
+
+```ts
+export interface TinyliciousContainerConfig {
+  id: string;
+  logger?: ITelemetryBaseLogger;
+}
+```
+`TinyliciousContainerConfig` interface definition takes an optional parameter `logger`. (The definition is similar to `FrsContainerConfig` interface)
+
+
+```ts
+public async createContainer(
+  serviceContainerConfig: TinyliciousContainerConfig,
+  containerSchema: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+`createContainer()` interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The `FrsClient` will take `FrsContainerConfig` for its respective methods)
+
+```ts
+static async getContainer(
+  serviceConfig: TinyliciousContainerConfig,
+  objectConfig: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+`getContainer()` interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The `FrsClient` will take `FrsContainerConfig` for its respective methods)
+
+
+```ts
+const loader = new Loader({
+  urlResolver: this.urlResolver,
+  documentServiceFactory: this.documentServiceFactory,
+  codeLoader,
+  logger: tinyliciousContainerConfig.logger,
+});
+```
+The `Loader` object is called by both `createContainer()` and `getContainer()`, and requires a `ILoaderProps` interface as its constructor argument. `ILoaderProps` interface has an optional logger parameter that will take the `ITelemetryBaseLogger` defined by the user.
+
+```ts
+constructor(loaderProps: ILoaderProps) {
+  const scope = { ...loaderProps.scope };
+  if (loaderProps.options?.provideScopeLoader !== false) {
+      scope.ILoader = this;
+  }
+
+  this.services = {
+      urlResolver: createCachedResolver(MultiUrlResolver.create(loaderProps.urlResolver)),
+      documentServiceFactory: MultiDocumentServiceFactory.create(loaderProps.documentServiceFactory),
+      codeLoader: loaderProps.codeLoader,
+      options: loaderProps.options ?? {},
+      scope,
+      subLogger: DebugLogger.mixinDebugLogger("fluid:telemetry", loaderProps.logger, { all:{loaderId: uuid()} }),
+      proxyLoaderFactories: loaderProps.proxyLoaderFactories ?? new Map<string, IProxyLoaderFactory>(),
+      detachedBlobStorage: loaderProps.detachedBlobStorage,
+  };
+  this.logger = ChildLogger.create(this.services.subLogger, "Loader");
+}
+```
```

Comment:
Like above, I would avoid pasting huge chunks of code in here that is totally subject to change.  Instead, if you want the reader to be able to dive in and see _how_ this all works, try permalinks to code, like this:  https://github.com/microsoft/FluidFramework/blob/main/packages/loader/container-loader/src/loader.ts#L309

(you can also do multi-line by manually doing something like #L1-L10 in the URL)

Reply:
Oooh I didn't know it would render like that in the comment.  The URL is just: `https://github.com/microsoft/FluidFramework/blob/24f5cd9acd3a7ef83604edd9863a47ea50c30569/packages/loader/container-loader/src/loader.ts#L309`

Reply:
Just click on a line number when viewing a file on github.com, then under the `...` do "Copy permalink"

Reply:
Did not know Github has permalink built in, been doing it manually 😅 Thanks!!

Reply:
Rather than linking to source code, I would link to the API docs, like the ones here: https://fluidframework.com/docs/apis/common-definitions/itelemetrybaselogger/